toolkit/tools/pkg/imagecustomizerlib/typeConversion.go

```diff
@@ -158,14 +158,51 @@ func mountIdentifierTypeToImager(mountIdentifierType imagecustomizerapi.MountIde
 		return configuration.MountIdentifierPartLabel, nil
 
 	default:
-		return "", fmt.Errorf("unknwon MountIdentifierType value (%s)", mountIdentifierType)
+		return "", fmt.Errorf("unknown MountIdentifierType value (%s)", mountIdentifierType)
 	}
 }
 
 func kernelCommandLineToImager(kernelCommandLine imagecustomizerapi.KernelCommandLine,
+	existingSELinuxMode imagecustomizerapi.SELinux,
 ) (configuration.KernelCommandLine, error) {
+	imagerSELinux, err := selinuxModeMaybeDefaultToImager(kernelCommandLine.SELinux, existingSELinuxMode)
+	if err != nil {
+		return configuration.KernelCommandLine{}, err
+	}
+
 	imagerKernelCommandLine := configuration.KernelCommandLine{
 		ExtraCommandLine: kernelCommandLine.ExtraCommandLine,
+		SELinux:          imagerSELinux,
+		SELinuxPolicy:    "",
 	}
 	return imagerKernelCommandLine, nil
 }
+
+func selinuxModeMaybeDefaultToImager(selinuxMode imagecustomizerapi.SELinux,
+	existingSELinuxMode imagecustomizerapi.SELinux,
+) (configuration.SELinux, error) {
+	if selinuxMode == imagecustomizerapi.SELinuxDefault {
+		selinuxMode = existingSELinuxMode
+	}
+
+	return selinuxModeToImager(selinuxMode)
+}
+
+func selinuxModeToImager(selinuxMode imagecustomizerapi.SELinux) (configuration.SELinux, error) {
```

Comment:
Suggestion: the `selinuxModeMaybeDefaultToImager` name feels a bit unclear. Since the main point is to get the `ToImager` value, what do you think about just moving the `if` into `selinuxModeToImager` and dropping this additional function?
```suggestion
func selinuxModeToImager(selinuxMode imagecustomizerapi.SELinux,
	existingSELinuxMode imagecustomizerapi.SELinux) (configuration.SELinux, error) {
	if selinuxMode == imagecustomizerapi.SELinuxDefault {
		return existingSELinuxMode, nil
	}
```

Reply:
`selinuxModeToImager` is used directly by the `handleSELinux()` function. In `handleSELinux()` function, the default value is already resolved. So, the `existingSELinuxMode` would just a dummy value, which I didn't like. Hence, why I split the functions.